cmake/configs/default.cmake

```diff
@@ -62,7 +62,7 @@ hunter_config(Libcxxabi VERSION ${HUNTER_Clang_VERSION})
 hunter_config(Libssh2 VERSION 1.7.0)
 hunter_config(Lua VERSION 5.3.2)
 hunter_config(MySQL-client VERSION 6.1.6)
-hunter_config(OpenCV VERSION 3.1.0-p1)
+hunter_config(OpenCV VERSION 3.1.0-p2 CMAKE_ARGS WITH_QT=YES)
```

Comment:
> My reasoning behind setting it by default

It should be set in `hunter.cmake` of OpenCV package:
* https://github.com/ruslo/hunter/wiki/usr.adding.new.package#cmake-options